gameboard/game.js

```diff
@@ -0,0 +1,93 @@
+let ships = [
```

Comment:
From what I can see, you don't modify this variable at all, so it should be const. 
Additionally, it should be named using the so-called `SCREAMING_SNAKE_CASE` notation.